Tests/FeatherDatabaseTests/_Utils/TestCase.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  File.swift
+//
+//
+//  Created by Tibor Bodecs on 16/02/2024.
+//
+
+import FeatherComponent
+import NIO
+import XCTest
+
+class TestCase: XCTestCase {
+
+    var eventLoopGroup: EventLoopGroup!
+    var components: ComponentRegistry!
+
+    override func setUp() async throws {
+        self.eventLoopGroup = MultiThreadedEventLoopGroup(numberOfThreads: 1)
```

Comment:
It's usually recommended to use `MultiThreadedEventLoopGroup.singleton` nowadays (which also doesn't have to be explicitly shut down).